lib/index.js

```diff
@@ -6,12 +6,9 @@
 var alias = require('alias');
 var dates = require('convert-dates');
 var del = require('obj-case').del;
-var each = require('each');
 var includes = require('includes');
 var integration = require('analytics.js-integration');
-var is = require('is');
 var iso = require('to-iso-string');
-var some = require('some');
```

Comment:
Are any of these dependencies unnecessary now? If so remove them from `component.json`